op-node/sources/types.go

```diff
@@ -83,71 +87,175 @@ func (info *HeaderInfo) ReceiptHash() common.Hash {
 	return info.receiptHash
 }
 
-type rpcHeaderCacheInfo struct {
+type rpcHeader struct {
+	ParentHash  common.Hash      `json:"parentHash"`
+	UncleHash   common.Hash      `json:"sha3Uncles"`
+	Coinbase    common.Address   `json:"miner"`
+	Root        common.Hash      `json:"stateRoot"`
+	TxHash      common.Hash      `json:"transactionsRoot"`
+	ReceiptHash common.Hash      `json:"receiptsRoot"`
+	Bloom       eth.Bytes256     `json:"logsBloom"`
+	Difficulty  hexutil.Big      `json:"difficulty"`
+	Number      hexutil.Uint64   `json:"number"`
+	GasLimit    hexutil.Uint64   `json:"gasLimit"`
+	GasUsed     hexutil.Uint64   `json:"gasUsed"`
+	Time        hexutil.Uint64   `json:"timestamp"`
+	Extra       hexutil.Bytes    `json:"extraData"`
+	MixDigest   common.Hash      `json:"mixHash"`
```

Comment:
Got confused between `MixDigest` and `mixHash`. Why not use a consistent name?

Reply:
`mixHash` is used in the JSON RPC + specs/EIPs, but `MixDigest` is used in the geth header/block types. And it may change to `PrevRandao` once the json field is finally renamed. Going to leave it consistent with geth for now.